src/Image/index.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { Image as AntImage, ImageProps as AntImageProps } from 'antd';
+import './style.css';
+
+export type ImageProps = AntImageProps;
+export const fallbackIcon =
```

Comment:
When using svg file in code, you can put them under `/assets/icons` as you did previously. The base64 format is needed when we render icons in css

Reply:
the `fallbackIcon` prop only accepts type `string` to be passed as the value, like a URL for example.